### ISSUE TITLE
Unable to install older version

### ISSUE THREAD

**Issue by TeodorKolev** [jbosstools/jbosstools-livereload#116]
Hi, How can I install older version, because I don't want to switch my Java version to latest

**Comment by xcoulon** [comment]
You can try to install LiveReload from an older JBoss Tools update site: http://download.jboss.org/jbosstools/updates/stable/luna/

**Comment by xcoulon** [comment]
@TeodorKolev but be aware that other JBoss Tools plugins also require Java 8.

**Comment by TeodorKolev** [comment]
I am with Mars 1 and that insructions doesn't work

**Comment by TeodorKolev** [comment]
So there is no way to run jboss livereload in Java 1.7???

**Comment by TeodorKolev** [comment]
Who is the man with big head who has decide this ?

**Comment by maxandersen** [comment]
@TeodorKolev  correct - this was announced back in June, see details on http://tools.jboss.org/blog/2015-06-23-beta1-for-mars.html.

**Comment by TeodorKolev** [comment]
it is not working...

**Comment by maxandersen** [comment]
What is not working ? Java 8 ?

**Comment by maxandersen** [comment]
If you want to use Java 7 I recommend you continue to use Eclipse Luna.

**Comment by TeodorKolev** [comment]
Eclipse mars 1 Java 1.7 jboss livereload installed but when I go to servers there is no such server engine as jboss like before in (luna)

**Comment by xcoulon** [comment]
Which server engine are you talking about ? Latest server tooling also requires Java 8. Which version do you have ?
Also, may I ask why installing Java 8 to run JBoss Tools is an issue ? You can still build and deploy on Java 7, whatever version of Java Eclipse runs on.

**Comment by TeodorKolev** [comment]
I do not want to install Java 8. Am I able to use jboss livereload on my Eclipse Mars 1 or not? If I can how to do that?

**Comment by robstryker** [comment]
No, you cannot. We suggest you use Eclipse Luna with previous versions of JBoss Tools if you want to use Java 7.

**Comment by TeodorKolev** [comment]
Why I cannot. Where is the logic? Why you build it like that to set max Java version as minimum??

**Comment by maxandersen** [comment]
@TeodorKolev the blog outlines the primary reasons, but I can outline more here:

a) Java 7 is no longer supported on the desktop - Java 8 is.
b) Parts of the Eclipse plugins we are relying on now requires Java 8 
c) Java 7 requires you to fiddle with permgen - Java 8 does not.
d) Installing Java 8 is trivial and can run side-by-side with Java 7
e) Java 7 is no longer receiving security updates - Java 8 does
f) Eclipse still can work with projects targeting Java 1,2,4,5,6,7,8 and 9 thus using Java 8 should not be a problem
h) Adoption of Java 8 on desktop is now well beyond 60% in our usage statistics
i) We asked community 6 months ago if any issues moving and got only positive feedback on moving. 
j) Using Java 8 lets us clean up the code to deliver features faster.
h) Eclipse Neon will require Java 8 100%

I'm truly interested in why you cannot run Java 8 on your machine ? If your only reason is "I don't want to" then please be okey with us also saying: We don't want to go back to Java 7, but you are welcome to use older versions. They are freely available for you to use.

If you have an *actual* technical or company policy reason then lets talk options.

Thank you.
